app-modules/authorization/tests/Feature/Filament/Resources/User/ViewUserTest.php

```diff
@@ -131,3 +131,190 @@
     expect($second->isImpersonated())->toBeTrue();
     expect(auth()->id())->toBe($second->id);
 });
+
+it('does not display the mfa_status Action for an external User', function () {
+    $user = User::factory()->create([
+        'is_external' => true,
+    ]);
+
+    asSuperAdmin();
+
+    livewire(ViewUser::class, [
+        'record' => $user->getRouteKey(),
+    ])
+        ->assertActionHidden('mfa_status');
+});
+
+it('displays the proper mfa_status Action for an internal User without MFA enabled', function () {
+    $user = User::factory()->create([
+        'is_external' => false,
+    ]);
```

Comment:
Factory users are not external by default.

```suggestion
    $user = User::factory()->create();
```

Reply:
Though I do see that, I just want to make sure I am being explicit here to figure proof the test.

Reply:
Then I recommend we create an `internal` UserFactory state function so we're not manipulating the state in one case and the attributes in another.

```php
public function internal(): static
{
    return $this->state(fn (array $attributes) => [
        'is_external' => false,
    ]);
}
```